consumers/writers/influxdb/README.md

```diff
@@ -12,12 +12,20 @@ default values.
 | ----------------------------- | --------------------------------------------------------------------------------- | ---------------------- |
 | MF_BROKER_URL                 | Message broker instance URL                                                       | nats://localhost:4222  |
 | MF_INFLUX_WRITER_LOG_LEVEL    | Log level for InfluxDB writer (debug, info, warn, error)                          | info                   |
-| MF_INFLUX_WRITER_PORT         | Service HTTP port                                                                 | 8180                   |
+| MF_INFLUX_WRITER_PORT         | Service HTTP port                                                                 | 8900                   |
 | MF_INFLUX_WRITER_DB_HOST      | InfluxDB host                                                                     | localhost              |
 | MF_INFLUXDB_PORT              | Default port of InfluxDB database                                                 | 8086                   |
 | MF_INFLUXDB_ADMIN_USER        | Default user of InfluxDB database                                                 | mainflux               |
 | MF_INFLUXDB_ADMIN_PASSWORD    | Default password of InfluxDB user                                                 | mainflux               |
 | MF_INFLUXDB_DB                | InfluxDB database name                                                            | mainflux               |
+| MF_INFLUXDB_HOST              | InfluxDB host name                                                                | mainflux-influxdb      |
+| MF_INFLUXDB_PROTOCOL          | InfluxDB protocol                                                                 | http                   |
+| MF_INFLUXDB_TIMEOUT           | InfluxDB timeout value                                                            | 1s                     |
```

Comment:
Improve descriptions - this is client connection readiness timeout. Also, since Reader is using the same internal client, Reader README probably needs an update, too.